server/helpers/runnerHelper.js

```diff
@@ -0,0 +1,122 @@
+const map = require('lodash/map');
+const forEach = require('lodash/forEach');
+const get = require('lodash/get');
+const includes = require('lodash/includes');
+const filter = require('lodash/filter');
+const find = require('lodash/find');
+const flatten = require('lodash/flatten');
+const compact = require('lodash/compact');
+const replace = require('lodash/replace');
+
+const { getProjectName } = require('./testrailHelper');
+const { getParsedTestCase } = require('./parserHelper');
+
+const getProjectsCommonParsedTests = data => {
+  const { projects, commonParsedTest } = data;
+  const suiteName = get(commonParsedTest, 'suiteName');
+  const currentProjectName = getProjectName(suiteName);
+  if (!includes(projects, currentProjectName)) {
+    return [];
+  }
+  const filteredProjects = filter(
+    projects,
+    project => project !== currentProjectName,
+  );
+  return map(filteredProjects, project => ({
+    ...commonParsedTest,
+    suiteName: replace(suiteName, new RegExp(currentProjectName, 'g'), project),
+  }));
+};
+
+const getRepoProjectCommonSuiteCases = data => {
+  const { parsedTests, projects } = data;
+  const commonParsedTests = filter(parsedTests, parsedTest =>
+    get(parsedTest, 'custom_is_common'),
+  );
+
+  const projectsCommonParsedTests = map(commonParsedTests, commonParsedTest =>
+    getProjectsCommonParsedTests({ commonParsedTest, projects }),
+  );
+  return flatten(projectsCommonParsedTests);
+};
+
+const getRepoProjectCommonSuiteNames = parsedTests =>
+  map(parsedTests, 'suiteName');
+
+const getParsedTests = data => {
+  const { suites } = data;
+
+  const tests = map(suites, suite =>
```

Comment:
return compact(reduce(describes, (acc, describe) => acc.concat(map(describe.tests, (it) => {
  const ! = getParsedTestCase({
      content: it.body,
      testName: it.title,
    });
    if (!parsedTest) {
     return null;
    }
    return { ...parsedTest, suiteName: describe.title };
})),[]))